src/instructions/Instructions.test.jsx

```diff
@@ -147,4 +147,74 @@ describe('SequenceExamWrapper', () => {
 
     expect(getByTestId('pending-prerequisites')).toBeInTheDocument();
   });
+
+  it('Instructions for error status', () => {
+    store.getState = () => ({
+      examState: {
+        isLoading: false,
+        proctoringSettings: {
+          link_urls: '',
+        },
+        verification: {
+          status: 'none',
+          can_verify: true,
+        },
+        activeAttempt: {
+          attempt_status: 'error',
+        },
+        exam: {
+          time_limit_mins: 30,
+          attempt: {
+            attempt_status: 'error',
+          },
+        },
+      },
+    });
+
+    render(
+      <ExamStateProvider>
+        <Instructions>
+          <div data-testid="sequence-content">Sequence</div>
+        </Instructions>
+      </ExamStateProvider>,
+      { store },
+    );
+    expect(screen.getByText('Error with proctored exam')).toBeInTheDocument();
+  });
+
+  it('Instructions for ready to resume status', () => {
+    store.getState = () => ({
+      examState: {
+        isLoading: false,
+        proctoringSettings: {
+          link_urls: '',
+          platform_name: 'Platform Name',
+        },
+        verification: {
+          status: 'none',
+          can_verify: true,
+        },
+        activeAttempt: {
+          attempt_status: 'ready_to_resume',
+        },
+        exam: {
+          time_limit_mins: 30,
+          attempt: {
+            attempt_status: 'ready_to_resume',
+          },
+        },
+      },
+    });
+
+    render(
+      <ExamStateProvider>
+        <Instructions>
+          <div data-testid="sequence-content">Sequence</div>
+        </Instructions>
+      </ExamStateProvider>,
+      { store },
+    );
+    expect(screen.getByText('Your exam is ready to be resumed.')).toBeInTheDocument();
```

Comment:
I think we should be testing for the correct "remaining time" value here. If you want to cover this in one of the test PRs that's fine by me